src/components/Menu/Menu.module.css

```diff
@@ -17,34 +17,35 @@
  */
 
 .menu {
+  --mp: calc(var(--spacing-padding-md, 12) * 1px);
+  --mp-sm: calc((var(--spacing-padding-md, 12) * 1px) / 2);
+
   :global([role*=menuitem]) {
     display: flex;
     align-items: center;
   }
 
-  ul[role*=menu] li div[role*=presentation] {
-    padding-left: calc(var(--spacing-margin-xs, 4) * 1px + var(--shape-size-lg, 24) * 1px) !important;
+  /* Inline */
+  &:global(.mia-platform-menu-inline) ul[role*=menu] {
```

Comment:
are the & necessary?

Reply:
yes. it means that when the menu class has inline sister class it applies this CSS